### PR TITLE
DL-13364 - Added basic page and controller

### PR DESCRIPTION
**Pull request by nehakannaujia1**
# JIRA Story or Github Number - descriptive title

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist

*Reviewee* (Replace with your name)
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

*Reviewer* (Replace with your name)
 - [ ]  I've confirmed that every effort has been made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've confirmed appropriate tests has been included with any code added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've confirmed code was added using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by nehakannaujia1** [comment]
Apologies , I deleted it accidentally, bringing in for review again.

File: app/controllers/propertyDetails/PropertyDetailsExitController.scala
Comment (benryan1790) [inline app/controllers/propertyDetails/PropertyDetailsExitController.scala:42]: This line is unnecessary, the template already has access to an implicit messages so we can remove this line of code

File: app/controllers/propertyDetails/PropertyDetailsExitController.scala
Comment (benryan1790) [inline app/controllers/propertyDetails/PropertyDetailsExitController.scala:45]: We should add in the check of the newRevaluedFeature flag in the config like we did in DL-13360, to ensure that users cannot mistakenly access this page before we are ready (should wait until DL-13360 is merged)

**Review by benryan1790 — CHANGES_REQUESTED** [review]

**Pull request merged**
